p2p/src/p2p_event.rs

```diff
@@ -180,3 +192,31 @@ impl std::fmt::Display for P2pChannelEvent {
         }
     }
 }
+
+impl fmt::Display for P2pDiscoveryEvent {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            Self::Ready => write!(f, "p2p discovery ready"),
```

Comment:
please use same format as in the rest of the events. This is important as this is used in testing framework to match events.